src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -50,6 +51,29 @@
 public class JavadocDetailNodeParser {
 
     /**
+     * Options for errors listeners attached to ANTLR parser.
+     */
+    public enum ErrorListenerOption {
```

Comment:
Team city was giving errors about `public method with boolean option`. Upon looking about it further, using an `enum` was recommended by the corresponding inspection.